starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,74 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+* {
+  font-family: Open Sans;
+}
+
+header {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.pieheader {
+  height: 800px;
+  margin: 0 auto;
+  background-image: url("../images/apple-pie.jpg");
+  background-size: 100%, 100%;
```

Comment:
to control background size to cover the dimensions of the div, you just have to put `background-size: cover;`. This image seemed to have been a little big. You can fix it by adjusting the height of the div.